pipeline.yaml

```diff
@@ -11,7 +11,7 @@
 # files are properly formatted.
 
 template: "regression/v1"
-# Specifies the dataset to use for model development
+# Specifies the datasets to use for model development and batch scoring
```

Comment:
The 'data' section is still only used for model development (training and evaluation). Why adding 'batch scoring' here?

Reply:
I'll make a separate comment for batch scoring....was trying to combine those comments but I see how this can be misleading.